pkg/txnutil/gc/testing.go

```diff
@@ -36,3 +36,5 @@ func (m *MockPDClient) UpdateServiceGCSafePoint(ctx context.Context, serviceID s
 func (m *MockPDClient) GetTS(ctx context.Context) (int64, int64, error) {
 	return oracle.GetPhysical(time.Now()), 0, nil
 }
+
+func (m *MockPDClient) Close() {}
```

Comment:
Why do we need an empty close method here?

Reply:
To implement PDClient interface.